### PR TITLE
updated pre-release release notes

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updated the Amazon Sales Channel pre-release release notes - changes marketplace link to coming soon and  comments out link, and it changes the release version from 1.0 to 2.0 and adds a note stating 1.0 for limited release only.

## Affected DevDocs pages

https://devdocs.magento.com/extensions/amazon-sales/release-notes/

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@hickskenh thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4429) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4429) before we can accept your contribution.<br/>

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: extensions/amazon-sales/release-notes/index.md [hunk 1]
```diff
@@ -9,17 +9,20 @@ See the following documentation:
...
 -   {:.new}New features
 -   {:.fix}Fixes and improvements
 -   {:.bug}Known issues
 
-### v1.0.0
+### v2.0.0
 
-Amazon Sales Channel 1.0.0 is compatible with version 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+Amazon Sales Channel 2.0.0 is compatible with version 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+**Note** Version 1.0 was available in limited release only.
```
Comment (lorikrell) [inline extensions/amazon-sales/release-notes/index.md]: This note may need to be in a note format. 
```suggestion

{:.bs-callout .bs-callout-info}
Version 1.0.0 was available in limited release only. 
```

Comment (jeff-matthews) [reply extensions/amazon-sales/release-notes/index.md]: Good catch Lori. We'll get that updated.

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Pull request merged**